src/main/java/com/github/krzysiekjodlowski/ox/HumanPlayer.java

```diff
@@ -0,0 +1,44 @@
+package com.github.krzysiekjodlowski.ox;
+
+import com.github.krzysiekjodlowski.ox.model.FieldNumber;
+import com.github.krzysiekjodlowski.ox.model.Symbol;
+import com.github.krzysiekjodlowski.ox.ui.UI;
+
+/**
+ * ${@inheritDoc} Human version.
+ *
+ * @author Krzysztof Jodlowski
+ */
+class HumanPlayer extends Player {
+
+  HumanPlayer(Symbol symbol) {
+    super(symbol);
+  }
+
+  /**
+   * Extends the one used in parent class.
+   *
+   * @return human Player representation
+   */
+  @Override
+  public String toString() {
+    return String.format("%s (human)", super.toString());
+  }
+
+  /**
+   * Collects human player input.
+   *
+   * @param ui generic user interface
+   * @return move made by player
+   * @throws NumberLowerThanOneException when want to mark field lower than 1
+   */
+  @Override
+  public Move makeMove(UI<String, Integer> ui) throws NumberLowerThanOneException {
+    NumberRange<Integer> boardBoundaryFields = NumberRange.of(
+        Settings.FIRST_FIELD_NUMBER,
+        Settings.INSTANCE.boardCapacity()
+    );
```

Comment:
Doeas it change with every move? Because based on names I would assume not (boardCapacity and firstField should be the same for every game). If I am right maybe consider moving this assigment to same static / final field or something?

Reply:
Gone:)